instructor/templates/instructor/preview.html

```diff
@@ -0,0 +1,371 @@
+<link type="text/css" href="/static/ui/gen/experiment_setup.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/mit706s14/gen/addmultipledialog.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui/gen/common.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/js/skins/lightgray/content.inline.min.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui_old/back_button.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/instructor/gen/instructor_dashboard.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/instructor/gen/instructor_facs.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/instructor/gen/instructor_western_blot.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/instructor/gen/instructor_select_technique.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ref_lib/rl.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui_old/gen/facs.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/decusability/gen/addmultipledialog.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/assignments/mit706s13/gen/instructions.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/mit706s13/gen/addmultipledialog.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui/gen/experiment_design.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/assignments/tufts/gen/instructions.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/assignments/mit706s14ps2/gen/instructions.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/coin-slider/coin-slider-styles.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/css/smoothness/jquery-ui.min.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui/gen/microscopy.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/css/jqdialog.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/instructor/gen/instructor_experiment_setup.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/assignments/decusability/gen/instructions.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui/gen/homepage.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui_old/gen/experiment.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/ui/gen/assignments.gss.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/starcellbio.app.css?_=1416084641" rel="Stylesheet"/>
+<link type="text/css" href="/static/css/jquery.multiselect.css?_=1416084641" rel="Stylesheet"/>
```

Comment:
yeah, so if we are going have this mess multiple times, at least drop it into a base template, or a template called `crazy_css_include.html` and include it